Fomo/AttractionType.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  AttractionType.swift
+//  Fomo
+//
+//  Created by Christian Deonier on 2/29/16.
+//  Copyright © 2016 TeamAwesome. All rights reserved.
+//
+
+import UIKit
+
+// Preferences are set by passing an array of AttractionTypes to endpoint
+class AttractionType: NSObject {
+    
+    var id: Int?
+    var name: String?
+    
```

Comment:
There is a list of finite options we're interested in. Google places have a pretty large list and some places fall under multiple tags.